Cargo.lock

```diff
@@ -1946,9 +1954,9 @@ dependencies = [
 
 [[package]]
 name = "rustls-platform-verifier-android"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Why are there irrelevant version bumps? I'm afraid this might interfere with what renovatebot is doing... `cargo rm` didn't generate this, right?

Reply:
`Cargo.lock` is automatically managed by Cargo... And this seems not a bump, because it only modified the patch number, according to semver.